src/pages/develop/cli.md

```diff
@@ -23,6 +23,24 @@ npm i -g @railway/cli
 sh -c "$(curl -sSL https://raw.githubusercontent.com/railwayapp/cli/master/install.sh)"
 ```
 
+**Windows (via Scoop)**
+
+*This should be used incase the npm method doesn't work for you.*
+
+**Note:** you might need to run this command in an administrative PowerShell instance to make Scoop work.
```

Comment:
Nit:

Change copy to: `Scoop requires all commands top be run in an administrative PowerShell instance.`

Reply:
I've changed it, because only that command requires admin perms.